### DISCUSSION TITLE
How to navigate throught the different tabs ?

### DISCUSSION THREAD

**Discussion by entrervotrenom** [AutomaApp/automa#323] [Q&A] [answered]
Hi !
Thank you so much @Kholid060 for this work, it save me so much time !!!

I want to do almost the same action on 10 tabs in parallele (there's just 10 differents buttons on the original tab and I want to load 10 times the same page because working on one tab at a time is a long process and the 9 over items are shifted up when one is managed (deleted in fact). Therefore I need to handle them at once in a row and then trigger a delay and a loop of the general workflow to start working on 10 new items)

How can i navigate through the created tabs ?

I've tested Execute workflow but it waits for the first workflow to end before starting the next one. I really need to switch tab. Once the job is done on the 10 tabs, I know how to repeat from the start.

**Comment by Kholid060** [comment, marked as answer]
I've been added the "Switch tab" block where you can switch between tabs, but this feature will be available on v1.0.0 (the next update).

**Reply by entrervotrenom** [reply]
Hi ! I tested "Switch tab". It works perfectly.

But i think there's a difference between 

1. going to the tab that has xxxxxx as an adresse 
2. going to the tab on the left or on the right from the current tab
3. going to the xth tab (like the first or the third or the 56324785th tab)

I explain myself with my case. I want to work with 10 times the same page opened in 10 tabs. And I want to do in each tab consecutively the same steps. 

Like : 

1. Action A in tab 1
2. Action A in tab 2
3. ...
4. Action A in tab 10
5. Action B in tab 1
6. Action B in tab 2
Etc...

Therefore I think I need the solution 2. or 3.
